### DISCUSSION TITLE
Make Ability To Edit Habit Data From a Long Time Ago Easier To Find

### DISCUSSION THREAD

**Discussion by landry314** [iSoron/uhabits#1294] [Feature Requests] [answered]
I have a list of habits I want to add over the last year but I cant seem to add a habit past a few weeks ago. I forgot to add them and just got a list.

Why cant we click on the calendar and add days for far in the past?

**Comment by dennys** [comment]
Maybe the following is not a good solution... 
I export the database into SD card, use a SQLite editor to add records into the file, and then import into App, it works.

This is a SQL sample and I use the [online converter](https://www.epochconverter.com/) to convert date to timestamp.
And you can check your habit id from 'Habits' table.
```
INSERT INTO "main"."Repetitions"
("habit", "timestamp", "value")
VALUES (4, 1640044800000, 3559000);
```

**Comment by neekubee** [comment, marked as answer]
In a particular habit, It's just below the calendar, "edit"

**Comment by landry314** [comment]
Ah, i found the Edit button, it was right there but not easy to see somehow. Maybe as a suggestion, it could be outlined with a box or somehow made more clear, like a bold font or something.

**Comment by landry314** [comment]
Weird that github doesn't put title changes inline in the discussion anymore. I just edited the title to my suggestion above.
